src/main/java/build/buildfarm/server/services/OperationsService.java

```diff
@@ -38,22 +38,25 @@ public OperationsService(Instance instance) {
   public void listOperations(
       ListOperationsRequest request, StreamObserver<ListOperationsResponse> responseObserver) {
     int pageSize = request.getPageSize();
-    if (pageSize < 0) {
+    if (pageSize < 0 || pageSize > 100) {
       responseObserver.onError(Status.OUT_OF_RANGE.asException());
       return;
     }
+    if (pageSize == 0) {
+      pageSize = 50;
```

Comment:
Consider refactoring this into a constant

Reply:
Done